test/mock-components/mock-invoice-number.js

```diff
@@ -2,5 +2,5 @@ const SFI_FIRST_PAYMENT = 'S0000001SFIP000001V001'
 
 module.exports = {
   SFI_FIRST_PAYMENT,
-  SFI_FIRST_PAYMENT_ORIGINAL: 'SFI000000'
+  SFI_FIRST_PAYMENT_ORIGINAL: 'SFI0000001'
```

Comment:
slice has changed back to `(1,8)` -- is this right?

Reply:
Yes, it is confirmed that invoice number should be 10 characters